airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -349,16 +349,19 @@ def create_job_yaml(self, context):
             'execution_date'].isoformat()
         instance_env[
             'AIRFLOW_ENABLE_XCOM_PICKLING'] = configuration.getboolean(
-                'core', 'enable_xcom_pickling')
+            'core', 'enable_xcom_pickling')
         instance_env['KUBERNETES_JOB_NAME'] = unique_job_name
-        instance_env['AIRFLOW_MYSQL_HOST'] = '127.0.0.1'
+        instance_env['AIRFLOW_MYSQL_HOST'] = configuration.get('mysql', 'host')
         instance_env['AIRFLOW_MYSQL_DB'] = configuration.get('mysql', 'db')
         instance_env['AIRFLOW_MYSQL_USERNAME'] = KubernetesSecretParameter(
             secret_key_name='airflow-cloudsql-db-credentials',
             secret_key_key='username')
         instance_env['AIRFLOW_MYSQL_PASSWORD'] = KubernetesSecretParameter(
             secret_key_name='airflow-cloudsql-db-credentials',
             secret_key_key='password')
+        instance_env['AIRFLOW_MYSQL_SSL_CERT_PATH'] = configuration.get('mysql', 'ssl_full_cert_path')
```

Comment:
should we handle the case where we are not using ssl here? if we do, we can deploy airflow now without that configuration and things will work as they do now...

Reply:
when will we not use ssl? This is where we decide what is used for XComs. XComs searches for AIRFLOW_MYSQL_DB/HOST, we set that here. We would need something to tell us when to know what to use. Since this code is deployed with Airflow, I am not sure how we would do that.

Reply:
We would somehow have to know based on the image if we should or shouldn't use ssl

Reply:
or are you saying if we deploy this and say if ssl_cert: add ssl configs
then we would also need if not ssl_cert: add another cloudsql port

Reply:
well actually that doesn't work because if ssl_cert will always be true because it comes from kube_airlfow